decred-dcrdex/README.md

```diff
@@ -1,27 +1,27 @@
-## Publishing the docker image
+# Publishing Docker image
 
-The `dcrdex` image can be built and published to Docker Hub using the following steps.  This requires the use of [BuildKit](https://docs.docker.com/build/buildkit/), which is part of recent docker releases.
+The `bisonwallet` image can be built and published to Docker Hub using the following steps.  This requires the use of [BuildKit](https://docs.docker.com/build/buildkit/), which is part of recent Docker releases.
 
-1. Log in to Docker Hub using the credentials that has write access to https://hub.docker.com/u/decred
+1. Log in to Docker Hub using the credentials that has write access to <https://hub.docker.com/u/decred>
```

Comment:
nit: maybe   "credentials that `have` write access"

Reply:
fixed, thx